src/data/themes/index.ts

```diff
@@ -1,22 +1,35 @@
-import type { ImageMetadata } from '@astrojs/image'
+import type { ImageMetadata } from '@astrojs/image/dist/vite-plugin-astro-image.js'
 import data from '../themes.json'
 import type { Link, Theme } from '../../types.js'
 
 interface ThemeData {
     title: string
     description: string
-    image: { src: string; alt: string }
+    fullDescription?: string
+    image: { src: string; alt: string; }
+    images?: { src: string; alt: string; }[]
     repoUrl: Link
```

Comment:
This should be optional now

Reply:
Oh yeah that's a little confusing, this is actually the type *after* data comes out of the `update-themes.mjs` script.  Its defaulted to an empty array there, but I'll go ahead and mark it as optional here just in case